Student/Student/Assignments/AssignmentDetails/AssignmentDetailsPresenter.swift

```diff
@@ -104,6 +104,9 @@ class AssignmentDetailsPresenter: PageViewLoggerPresenterProtocol {
         self.assignmentID = assignmentID
         self.fragment = fragment
         self.submissionButtonPresenter = SubmissionButtonPresenter(env: env, view: view, assignmentID: assignmentID)
+        if let session = Keychain.mostRecentSession {
```

Comment:
I think `env.currentSession` is probably the better thing to look at than the `mostRecentSession`.